pkg/cloud/gen/main.go

```diff
@@ -1271,12 +1272,14 @@ const location = "location"
 `
 	tmpl := template.Must(template.New("header").Parse(text))
 	values := map[string]string{
-		"Year":                fmt.Sprintf("%v", time.Now().Year()),
-		"FilterPackage":       filterPackage,
-		"MetaPackage":         metaPackage,
-		"AlphaComputePackage": alphaComputePackage,
-		"BetaComputePackage":  betaComputePackage,
-		"GaComputePackage":    gaComputePackage,
+		"Year":                       fmt.Sprintf("%v", time.Now().Year()),
+		"FilterPackage":              filterPackage,
+		"MetaPackage":                metaPackage,
+		"AlphaComputePackage":        alphaComputePackage,
+		"BetaComputePackage":         betaComputePackage,
+		"GaComputePackage":           gaComputePackage,
+		"GANetworkservicesPackage":   gaNetworkServicesPackage,
+		"BetaNetworkservicesPackage": betaNetworkServicesPackage,
```

Comment:
Tiny nits: "Ga" vs "GA" is inconsistent here. Ordering also doesn't match (alpha > beta > ga followed by ga > beta)

Reply:
Done